src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -464,6 +468,10 @@ export class CheckoutPageComponent extends Component {
                   paymentInfo={intl.formatMessage({ id: 'CheckoutPage.paymentInfo' })}
                   authorDisplayName={currentAuthor.attributes.profile.displayName}
                   showInitialMessageInput={showInitialMessageInput}
+                  onCreateStripePaymentToken={this.props.onCreateStripePaymentToken}
```

Comment:
You should just spread this `onCreateStripePaymentToken ` out of props similarly as the other ones.